apps/chat/src/components/Common/AgentAndToolsetSelector/AgentAndToolsetModal.tsx

```diff
@@ -45,7 +51,7 @@ import {
   AgentAndToolsetSelectItemProps,
 } from './AgentAndToolsetSelectItem';
 
-export type EntityType = 'agents' | 'toolsets';
+import { orderBy } from 'lodash-es';
```

Comment:
```suggestion
import orderBy from 'lodash-es/orderBy';
```

There is some tree shaking problems inside lodash-es. Import utils this way